contrib/test_jenkins.sh

```diff
@@ -405,6 +405,11 @@ run_hello() {
 # Compile and run UCP hello world example
 #
 run_ucp_hello() {
+	if ./src/tools/info/ucx_info -e -u twe|grep ERROR
```

Comment:
is the last 'e' needed? I get no valid output when i run it...

Reply:
yes, part of this PR is to add support for 'e' which means error handling

Reply:
oh you added it, i see now :)